Gruntfile.js

```diff
@@ -62,6 +62,9 @@ module.exports = function(grunt) {
       customRegex: [
          '\{\{\\s*(?:::)?\'((?:\\\\.|[^\'\\\\])*)\'\\s*\\|\\s*translate(:.*?)?\\s*(?:\\s*\\|\\s*[a-zA-Z]*)?\}\}',
          '="\'((?:\\\\.|[^\'\\\\])*)\'\\s*\\|\\s*translate"',
+         'translate="([A-Z0-9\.\-\_]*)"',
+         'proofDocument.error = \'([A-Z0-9\.\-\_]*)\.', // handle values in preferenceProofUploader
```

Comment:
update these 2 pieces to use the t() notation to flag it for grunt, rename the t() function to make it a little clearer what it's doing